gf180mcu/openlane/gf180mcu_fd_sc_mcu7t5v0/config.tcl

```diff
@@ -11,14 +11,14 @@ set ::env(FP_WELLTAP_CELL) "$::env(STD_CELL_LIBRARY)__filltie"
 set ::env(FP_ENDCAP_CELL) "$::env(STD_CELL_LIBRARY)__endcap"
 
 # defaults (can be overridden by designs):
-set ::env(SYNTH_DRIVING_CELL) "$::env(STD_CELL_LIBRARY)__inv_1"
-set ::env(SYNTH_DRIVING_CELL_PIN) "ZN"
-set ::env(SYNTH_CLK_DRIVING_CELL) "$::env(STD_CELL_LIBRARY)__inv_4"
-set ::env(SYNTH_CLK_DRIVING_CELL_PIN) "ZN"
+set ::env(SYNTH_DRIVING_CELL) "$::env(STD_CELL_LIBRARY)__buf_4"
+set ::env(SYNTH_DRIVING_CELL_PIN) "Z"
+set ::env(SYNTH_CLK_DRIVING_CELL) "$::env(STD_CELL_LIBRARY)__buf_4"
+set ::env(SYNTH_CLK_DRIVING_CELL_PIN) "Z"
 
 # update these
-set ::env(SYNTH_CAP_LOAD) "72.91" ; # femtofarad from pin I in liberty file
-set ::env(SYNTH_MIN_BUF_PORT) "$::env(STD_CELL_LIBRARY)__buf_1 I Z"
+set ::env(SYNTH_CAP_LOAD) "9.096" ; # femtofarad from pin I in liberty file
```

Comment:
Not sure where you got these values from, seems like it's too low. Can you please point me to how you got this value?

Reply:
As per ORFS [config.mk](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/710a6af4e2934ad968e3967e5ae99707206e7e1a/flow/platforms/gf180/config.mk#L50) for 9track `export ABC_LOAD_IN_FF                         = 0.01343` 
For 7track *_buf_4 `capacitance : 0.009315 ;` . Its should be `9.315` as OL flow converting to FF

Reply:
1. The values in ORFS is different
2. Where did you get the value in ORFS?

Reply:
`libs.ref/gf180mcu_fd_sc_mcu9t5v0/liberty/gf180mcu_fd_sc_mcu9t5v0__tt_025C_5v00.lib`